### ISSUE TITLE
implement `clear`

### ISSUE THREAD

**Issue by prsabahrami** [prefix-dev/shell#168]
I'm not sure how we can implement this but I tried `clear` on shell on a Windows terminal and was not working.

**Comment by certik** [comment]
Yes, I was missing `clear` as well, for creating good looking screenshots.

**Comment by wolfv** [comment]
Fixed in #227
